packages/cli/index.js

```diff
@@ -24,10 +24,6 @@ const liveServer = require('./src/lib/live-server');
 const cliUtil = require('./src/util/cliUtil');
 const logger = require('./src/util/logger');
 
-const {
-  ACCEPTED_COMMANDS,
-  ACCEPTED_COMMANDS_ALIAS,
```

Comment:
Since `ACCEPTED_COMMANDS` and `ACCEPTED_COMMANDS_ALIAS` constants were only used here, what do you think of removing the `contants.js` file since it is no longer in use?

Reply:
Hi @jonahtanjz , I think `constants.js` has already been removed in this PR:)

Reply:
Oops 😬 Sorry missed it 😅